modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileIntegrationTest.java

```diff
@@ -146,37 +146,16 @@ public void setUp() throws Exception {
 			SamlSpSessionLocalServiceUtil.class,
 			SamlSpSessionLocalService.class);
 
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "credentialResolver", credentialResolver);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "localEntityManager",
-			keyStoreLocalEntityManager);
-		ReflectionTestUtil.setFieldValue(_webSsoProfileImpl, "portal", portal);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "samlBindingProvider", samlBindingProvider);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "samlProviderConfigurationHelper",
-			samlProviderConfigurationHelper);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "_samlSpAuthRequestLocalService",
-			_samlSpAuthRequestLocalService);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "samlSpSessionLocalService",
-			_samlSpSessionLocalService);
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "_samlSpMessageLocalService",
-			getMockPortletService(
-				SamlSpMessageLocalServiceUtil.class,
-				SamlSpMessageLocalService.class));
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "samlSpIdpConnectionLocalService",
-			getMockPortletService(
-				SamlSpIdpConnectionLocalServiceUtil.class,
-				SamlSpIdpConnectionLocalService.class));
-		ReflectionTestUtil.setFieldValue(
-			_webSsoProfileImpl, "_userLocalService",
-			getMockPortletService(
-				UserLocalServiceUtil.class, UserLocalService.class));
+		_setUpCredentialsResolver();
```

Comment:
Hey @jgarcialr, could we sort without create new `_setUp*` methods?

Reply:
Hi @rafaprax ,
Yes we can do it without the methods, I saw the methods in other tests and makes sense as it seems more naturally sorted.
Regarding the sorting without methods, the pattern is underscore variables and then without the underscore or the other way around?

I mean
_userLocalService
samlSpIdpConnectionLocalService 

or

samlSPIdpConnectionLocalService
_userLocalService 
?